packages/fxa-content-server/tests/functional/settings_v2/navigation.js

```diff
@@ -105,16 +105,6 @@ describe('navigation', () => {
   });
 
   it('can sign out', async ({ remote }) => {
-    await click(
-      selectors.SETTINGS_V2.AVATAR_DROP_DOWN_MENU.MENU_BUTTON,
-      selectors.SETTINGS_V2.AVATAR_DROP_DOWN_MENU.DISPLAY_NAME_LABEL,
-      remote
-    );
-    await click(
-      selectors.SETTINGS_V2.AVATAR_DROP_DOWN_MENU.SIGNOUT_BUTTON,
-      selectors.ENTER_EMAIL.HEADER,
-      remote
-    );
-    await testElementExists(selectors.ENTER_EMAIL.HEADER, remote);
+    await signOut(remote);
```

Comment:
Do you find the await syntax easier to follow than chained promises? At some point we can convert everything to use the same pattern.

Reply:
Great question! I honestly haven't thought about it much, but it would be nice to standardize on just one approach.